client/layout/index.jsx

```diff
@@ -167,6 +168,7 @@ Layout = React.createClass( {
 					isActive={ translator.isActivated() } />
 				{ this.renderPreview() }
 				{ config.isEnabled( 'happychat' ) && this.props.chatIsOpen && <AsyncLoad require="components/happychat" /> }
+				<AppBanner />
```

Comment:
I went down that route at first because it was easier and cleaner, but then run into a couple of issues. First, when I inserted it in Stats section for example, the fixed positioning was not working properly because of the scroll (the banner ends up on the bottom of the page). I'm not sure if it can be worked around by adding some additional containers, but the prospect of trying/doing that seemed messy and it might cause visual glitches in general layout.
Secondly, we need to show it when we open notifications, so it can be shown on every section due to that (or were you thinking about introducing it in Notifications client too?).